buildUtils.js

```diff
@@ -0,0 +1,121 @@
+import { generateSDAssets } from "./styledictionary.js";
+import { processAndWriteColors } from "./processColors.js";
+import { processAndWriteSemanticAndComponentTokens } from "./processTokens.js";
+import { zip } from "zip-a-folder";
+import fs from "fs-extra";
+import ora from "ora";
+import prompts from "prompts";
+import "dotenv/config";
+
+const ENV_PATH = "./.env";
+
+const spinner = ora();
+
+export async function fetchAndTransformTokens({
+  figmaProjectId,
+  outputFilePath,
+  tokenVariableCollection,
+  isDataviz = false,
+}) {
+  spinner.start("Reading Figma access token");
+
+  let token = process.env.FIGMA_TOKEN;
+
+  if (token) {
+    spinner.succeed("Using Figma access token from environment viariables");
```

Comment:
found a tiny typo :) 
```suggestion
    spinner.succeed("Using Figma access token from environment variables");
```